webpack.config.js

```diff
@@ -29,7 +29,7 @@ module.exports = {
         use: ["style-loader", "css-loader", "sass-loader"],
       },
       {
-        test: /\.css$/,
+        test: /\.(|scss|sass|css)$/,
```

Comment:
we had this to handle other files
```

      {
        test: /\.s[ac]ss$/i,
        use: ["style-loader", "css-loader", "sass-loader"],
      },
```

are you have trouble loading .css files or sass?

Reply:
Yes! For some reasons, I wasn't able to run build script due to the .scss issues. So, I added that.